src/rpc.rs

```diff
@@ -126,24 +144,42 @@ impl RpcSol for RpcSolImpl {
     fn get_transaction_count(&self, meta: Self::Metadata) -> Result<u64> {
         meta.request_processor.get_transaction_count()
     }
-    // fn send_transaction(&self, meta: Self::Metadata, to: String, tokens: i64) -> Result<String> {
-    //     let client_keypair = read_keypair(&meta.keypair_location.unwrap()).unwrap();
-    //     let mut client = mk_client(&meta.leader.unwrap());
-    //     let last_id = client.get_last_id();
-    //     let to_pubkey_vec = bs58::decode(to)
-    //         .into_vec()
-    //         .expect("base58-encoded public key");
-    //
-    //     if to_pubkey_vec.len() != mem::size_of::<Pubkey>() {
-    //         Err(Error::invalid_request())
-    //     } else {
-    //         let to_pubkey = Pubkey::new(&to_pubkey_vec);
-    //         let signature = client
-    //             .transfer(tokens, &client_keypair, to_pubkey, &last_id)
-    //             .unwrap();
-    //         Ok(bs58::encode(signature).into_string())
-    //     }
-    // }
+    fn request_airdrop(&self, meta: Self::Metadata, id: String, tokens: u64) -> Result<bool> {
```

Comment:
Use `Result<()>` if `return Ok(false)` is meaningless and unused.

Reply:
If I make that change, the RPC response will look like this: `{"jsonrpc":"2.0","result":null,"id":1}`
That null seems wonky to me.

Reply:
hmm, yeah `"result": true` would be nicer.  "I successfully requested an airdrop for you"

Reply:
Sure, `Result<bool>` then.